decidim/decidim
# Use `stylelint-prettier` rules with stylelint

#### :tophat: What? Why?
After #10669 we no longer get the error lens inline errors in VSCode for the SCSS files which I personally found very useful in order to fix these issues on the fly and saving time in the development process.

Through `stylelint-prettier` package, we can introduce the prettier rules to stylelint which would help this. This also makes the "Format SCSS files" step in the CI pipeline unnecessary because the stylelint step should already spot these issues.

#### :pushpin: Related Issues
- Related to #10669

#### Testing
1. Create an SCSS (e.g. remove a space before the opening brace `{`)
2. Run `npm run stylelint`
3. See the prettier error printed out

### :camera: Screenshots
![Error lens output in VSCode](https://user-images.githubusercontent.com/864340/232234246-8ee8c465-d4ef-488d-a191-63dd1c41af82.png)
